tcpb/utils.py

```diff
@@ -121,6 +169,8 @@ def job_output_to_atomic_result(
     atomic_result.extras.update(
         {
             "qcvars": {
+                "energy": jo_dict.get("energy"),
+                "gradient": jo_dict.get("gradient"),
```

Comment:
Take a look at the full results specification at: https://github.com/MolSSI/QCElemental/blob/ec9d754414824e129729f477e6077a14d859dc85/qcelemental/models/results.py. Throwing things into `qcvars` should be a last result if there isn't a fully structured way to specify a value. I've likely overused the `qcvars` field because I didn't know what half these values meant, but I'd like to migrate value to their proper field as I learn what they actually are and where they actually belong on an `AtomicResult` object.

Reply:
That is to say, if there's a better field for this (and I know one field is the `result.return_value`if `gradient` was the requested computation, as per: https://github.com/MolSSI/QCElemental/blob/ec9d754414824e129729f477e6077a14d859dc85/qcelemental/models/results.py#L555-L558), let's put it there! However, if you are trying to return a gradient when this wasn't the primary computation requested maybe `qcvars` is all we got??